network/manager.go

```diff
@@ -269,7 +307,9 @@ func (nm *networkManager) CreateNetwork(nwInfo *NetworkInfo) error {
 	if err != nil {
 		return err
 	}
-
+	if nm.IsStatelessCNIMode() {
```

Comment:
can we move this check inside nm.save() instead of repeating in multiple places?

Reply:
addressed in the next commit.